packages/entity-server/src/routes/hierarchy/middleware/filter.ts

```diff
@@ -73,6 +73,15 @@ const operatorToSqlComparator = {
   '>': '>' as const, // Greater than
   '>=': '>=' as const, // Greater than or equal
 };
+const operatorToSqlArrayComparator = {
+  '==': 'IN' as const, // Contained in array
```

Comment:
Technically this isn't used since the `convertValueToAdvancedCriteria` exits early for `==`, and the default behaviour is to use `IN` if we use an array value without a specific sql comparator.

I've left it here anyway so if we change the logic elsewhere it's specifically defined